Lesson Plans/Flight Instructor/CFI/CFI1.1 Human Behavior.md

```diff
@@ -0,0 +1,126 @@
+			1. This can also manifest as a mismatch of pace: Slow pace applied to a fast, motivated learner
+	3. **Worry or Lack of Interest**: Learners concerns about their progress in the training; personal problems, dislike of training program, instructor or even completely unrelated reasons
+		1. The instructor can adapt by diverting their attention from the worries and focus on the task at hand.
+		2. ... but more importantly, the instructor should prevent it by making sure learners understand exactly where they are in the process, how well they progress and their deficiencies.
+	4. **Physical Discomfort, Illness, Fatigue, and Dehydration**
+		1. Airsickness: No sure cure; but tolerance might be built up over time. Instruction should stop when it manifests.
+		2. Fatigue: one of the most treacherous hazards to flight safety — it may not be apparent until a serious mistake is made
+			1. Acute fatigue: Normal occurrence of everyday living
+			2. Chronic fatigue: Combination of both physiological problems and psychological issues (ex. financial, home life, job stress, etc.)
+			3. If detected, a break in instruction and practice should be given or pacing should be adjusted.
+		3. Dehydration: Critical loss of water from body. It reduce a pilot's level of alertness. *Be alert in hot summer days and high altitude. Carry water on every flight.*
+		4. Heatstroke: Inability of the body to control its temperature
+	5. **Apathy Due to Inadequate Instruction**
+		1. Instructor needs to provide well-planned, appropriate and accurate instruction and adjusted to the learner's level
+		2. *Once the instructor loses learner confidence, it is difficult to regain.*
+	6. **Stress**
+		1. Normal Reaction
+			1. Respond rapidly and exactly, within their limits of experience and training.
+			2. This is desirable to a certain degree as one tend to think rationally and act rapidly (but relies on automatic response for many responses.)
+		2. Abnormal Reaction
+			1. Some response can be absent, inadequate, random or illogical.
+			2. Inappropriate reactions: eg. extreme over-cooperation, painstaking self-control, inappropriate laughing or singing and very rapid change in emotions
+			3. Marked changes in mood on different lessons: such as excellent morale followed by deep depression
+			4. Severe anger towards the instructor or others
+			5. *In a serious situation, instructors might need to carefully examine learners' response and their own response towards learners as it may leads to the following section ...*
+	7. **Seriously Abnormal Learners**
+		1. If the instructor believes the learner is suffering from a serious psychological abnormality:
+			1. Get a third-party instructor to conduct an evaluation flight and two instructors should confer to see if they agree further investigation is warranted
+			2. Legal responsibility: If the learner have serious psychological abnormality, solo flight endorsement / practice test recommendation should be withheld.
+
```

Comment:
Maybe some examples to talk through at the end of the lesson? Like on the [[Safety Pilot]] Page at the end